js/_website/src/app.html

```diff
@@ -4,6 +4,15 @@
 		<meta charset="utf-8" />
 		<link rel="icon" href="%sveltekit.assets%/favicon.png" />
 		<meta name="viewport" content="width=device-width" />
+		<script>
+			// prevents the flash of light mode by applying theme before page renders
+			(function () {
+				const theme = localStorage.getItem("theme") || "light";
```

Comment:
We should do a match media query here as a fallback instead of defaulting to light.